lib/matplotlib/__init__.py

```diff
@@ -1476,7 +1476,6 @@ def _jupyter_nbextension_paths():
 default_test_modules = [
     'matplotlib.tests.test_png',
     'matplotlib.tests.test_units',
-    'matplotlib.tests.test_widgets',
```

Comment:
So we *already* broke backcompat on our tests by having a requirement on pytest, and by making it not possible to run the test suite by running `nosetests`.  I simply do not understand why we can't just break it a bit more to get rid of old warts... that we're planning to get rid of anyways.